shared/src/main/scala/com/nawforce/pkgforce/workspace/Workspace.scala

```diff
@@ -117,28 +109,30 @@ object Workspace {
     path: PathLike,
     logger: CatchingLogger
   ): (Option[Workspace], IssuesManager) = {
-    val config             = loadWorkspaceConfig(path, logger)
-    val externalPathFilter = createExternalPathFilter(config)
+    val project            = loadSFDXProject(path, logger)
+    val externalPathFilter = createExternalPathFilter(project)
     val issueManager       = new IssuesManager(externalPathFilter)
     logger.issues.foreach(issueManager.add)
-    (Workspace(config, issueManager), issueManager)
+    (Workspace(project, issueManager), issueManager)
   }
 
-  private def loadWorkspaceConfig(
-    path: PathLike,
-    logger: CatchingLogger
-  ): Option[WorkspaceConfig] = {
+  private def loadSFDXProject(path: PathLike, logger: CatchingLogger): Option[SFDXProject] = {
     if (path.join("sfdx-project.json").exists) {
-      SFDXProject(path, logger).map(p => new SFDXWorkspaceConfig(path, p))
+      SFDXProject(path, logger)
```

Comment:
I assume since it works, this apply returns an Option. Should it log the same error if it returns `None`? like the other case.

If it already does this inside `SFDXProject` (logs other errors), then perhaps the remainder of this method should be moved there.

Reply:
Errors/Warnings get logged by SFDXProject, have added a comment to explain